themes/defaut/css/theme.css

```diff
@@ -75,26 +78,26 @@ ul.menu {}
 	margin-left: .5rem;
 }
 
-.menu>li {
+.menu a {
 	border-bottom: 3px solid transparent;
 }
-.menu>li:hover {
+.menu a:hover {
 	border-bottom: 3px solid #222;
 }
 .menu>li>span {
 	cursor: default;
 }
 .menu>li ul {/* sub menu */
-	display: none;
```

Comment:
For sub menu group rework (show on hover) on mobile, replace & change this to _.menu>li ul_
```
	display: block;
```
& line 99 it's not to **show** but **hide** _sub menu_ ;)
```
.menu>li:not(:hover) ul {/* hide sub menu */
```
Good job & idea  :+1: